modules/oe_content_event/config/install/field.field.node.oe_event.field_report_text.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_report_text
+    - node.type.oe_event
+  module:
+    - text
+id: node.oe_event.field_report_text
+field_name: field_report_text
+entity_type: node
+bundle: oe_event
+label: 'Report text'
+description: ''
+required: false
```

Comment:
Should be required.

Reply:
Done through constraint. By documentation this field is required only if any fields within Report fields group filled in.